site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -13,9 +13,22 @@ A match pattern is a URL with the following structure, used to specify a group o
 &lt;scheme&gt;://&lt;host&gt;/&lt;path&gt;
 ```
 
-**scheme**: Must be one of `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
-  - For information on injecting content scripts into unsupported schemes, such as `about:` and `data:`, see [Injecting in related frames][cs-frames].
-  - The scheme must be separated from the rest of the pattern using a double slash (`//`).
+**scheme**: Must be one of the following, separated from the rest of the pattern using a double slash (`//`):
+
+- `'http'`
+- `'https'`
+- A wildcard `*`, which matches only `http` or `https`
+- `'file'`
+- `'ftp'`
+- `'chromeui'`
+- `'extension'`
+- `'filesystem'`
+- `'ws'`
+- `'wss'`
+- `'data'`
```

Comment:
I'm a bit uncomfortable adding all these scheme's because I know that extensions have restrictions. For example, what is the 'extension' scheme? If it's `chrome-extension://` URLs, these are not valid for content script injection.

Reply:
This list came from source code: https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/url_pattern.cc;l=31-52;drc=8c08dd32bf007ca09f57b63b6b812478b44ac2c4

You're probably right about extension UI. (And a few other probably need to be spelled differently.) But confirm that in code yet.

Reply:
Notes:

- Chrome no longer supports FTP so I don't think `ftp` makes sense.
- I'm not familiar with (and couldn't find) any use case for `chromeui`. It seems internal so I would remove.
- The same for `extension` and `filesystem`.
- Looking at the code, we don't seem to actually support `data` in any APIs, so I would remove that.
- `uuid_in_package` was removed here: https://groups.google.com/a/chromium.org/g/blink-dev/c/iyhes6tMz2Y

So I think the full list should be:

```
http
https
A wildcard *, which matches only http or https
file
```

Reply:
+1